packages/ts/react-crud/src/autogrid-columns.tsx

```diff
@@ -112,6 +112,14 @@ export function getColumnOptions(
   // TODO: Remove eslint-disable when all TypeScript version issues are resolved
   // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
   return customColumnOptions
-    ? { ...typeColumnOptions, headerFilterRenderer, ...customColumnOptions }
+    ? {
+        ...typeColumnOptions,
+        headerFilterRenderer,
+        ...customColumnOptions,
+        // Use default renderer for column of type object, if custom path is set and no other renderer is configured
+        ...(propertyInfo.type === 'object' &&
+          customColumnOptions.path &&
+          customColumnOptions.renderer === undefined && { renderer: null }),
```

Comment:
Functionality wise this looks fine.

Nit: Though, I would prefer that the `getTypeColumnOptions` could have the proper info to not return the `AutoGridJsonRenderer` at all in case no custom renderer is defined and the `customColumnOptions?.path !== undefined`.

Reply:
@Lodin WDYT?

Reply:
I believe your comment on the issue could be another feature. Here, I simply think that the `getTypeColumnOptions` should actually return the correct renderer based on the `propertyInfo` and `customColumnOptions` so that it doesn't need to be patched later down the stream. Having the logic that decides on the renderer isolated in some method is important for future maintenance.

Reply:
What I'm suggesting is to have this taken care of in `getTypeColumnOptions` by adding the `columnOptions` as the parameter to it:
```ts
function getTypeColumnOptions(propertyInfo: PropertyInfo, customColumnOptions?: ColumnOptions)
```
and change the `case 'object':` e.g. as follows:
```tsx
case 'object':
    return {
        autoWidth: true,
        renderer:
          customColumnOptions?.path !== undefined && customColumnOptions.renderer == null ? null : AutoGridJsonRenderer,
        headerFilterRenderer: NoHeaderFilter,
    };
```